src/Form/ExerciseType.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Exercise;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use App\Entity\MuscleGroup;
+
+class ExerciseType extends AbstractType
```

Comment:
Це не обов'зково, але за можливості краще додавати до назви класу суфікс, який робить його назву більш очевидною і при використанні його в інших місцях дає можливість швидко зрозуміти його призначення не перевіряючи повних шлях в імпортах тощо.

```suggestion
class ExerciseFormType extends AbstractType
```